src/amp-story-player/amp-story-player-impl.js

```diff
@@ -350,15 +350,17 @@ export class AmpStoryPlayer {
   initializeStories_() {
     const anchorEls = toArray(this.element_.querySelectorAll('a'));
 
-    this.stories_ = anchorEls.map(
-      (anchorEl) =>
-        /** @type {!StoryDef} */ ({
-          href: anchorEl.href,
-          title: (anchorEl.textContent && anchorEl.textContent.trim()) || null,
-          poster: anchorEl.getAttribute('data-poster-portrait-src'),
-          iframeIdx: -1,
-        })
-    );
+    this.stories_ = anchorEls.map((anchorEl) => {
+      const img = anchorEl.querySelector('img[amp-story-player-image]');
```

Comment:
Should we get the `src` of the first `img`?
`amp-story-player-image` is an invalid attribute name, and we don't have validation to ensure publishers add the attribute.

Reply:
I think @newmuis wanted us to include an attribute in the poster image for future proofing https://github.com/ampproject/amphtml/issues/30512#issuecomment-703995772

Reply:
Sounds good, let's prefix it with `data-` then to make it valid HTML.